arrow-libs/optics/arrow-optics/api/arrow-optics.api

```diff
@@ -100,7 +104,7 @@ public abstract interface class arrow/optics/Getter : arrow/optics/Fold {
 	public abstract fun choice (Larrow/optics/Getter;)Larrow/optics/Getter;
 	public abstract fun compose (Larrow/optics/Getter;)Larrow/optics/Getter;
 	public abstract fun first ()Larrow/optics/Getter;
-	public abstract fun foldMap (Larrow/typeclasses/Monoid;Ljava/lang/Object;Lkotlin/jvm/functions/Function1;)Ljava/lang/Object;
```

Comment:
Unfortunately the change of the `empty / combine` method being the "main" one in `Fold` is breaking binary compatibility. But I don't know how to make these changes otherwise; any help is welcome.